### PR TITLE
README minor updates

### PR DESCRIPTION
**Pull request by olegbespalov**
# What?

Minor README fixes

# Why?

Just to give more details for the customers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,62 @@
...
+
+// A `stream.on` method adds a new handler for different kind of the events.
+// 
+// Currently supported: 
+// `data`  - triggered when the server send data to the stream
+// `error` - an error occurs
+// `end`   - triggered when the server has finished sending the data
+// You could register multiple handlers for the same kind event.
+stream.on('data', message => {
+  // server send data, processing...
```
Comment (codebien) [inline README.md:28]:
```suggestion
  // server send data, processing...
```
nit: I think a concrete example of how to use the message could be helpful.

**Review by codebien — COMMENTED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -2,8 +2,62 @@
 
 This extension is re-designing of the k6's GRPC module that also should bring new features like GRPC streaming.
 
+The extension code copied from the original k6's GRPC module, so you could use [an original documentation](https://k6.io/docs/javascript-api/k6-net-grpc/).
+
+The new stream's functionality:
```
Comment (olegbespalov) [inline README.md:7]:
```suggestion
The new stream's functionality (more examples you can find in the `examples` folder):
```

File: README.md (hunk 1 above)
Comment (codebien) [reply README.md:28]: Yeah, good idea.

**Review by mstoykov — APPROVED** [review]

**Pull request merged**